mne/evoked.py

```diff
@@ -414,18 +414,20 @@ def plot_topo(self, layout=None, layout_scale=0.945, color=None,
                   border='none', ylim=None, scalings=None, title=None,
                   proj=False, vline=[0.0], fig_background=None,
                   merge_grads=False, legend=True, axes=None,
-                  background_color='w', noise_cov=None, show=True):
+                  background_color='w', noise_cov=None, exclude=('bads'),
```

Comment:
my bad it should be
```suggestion
                  background_color='w', noise_cov=None, exclude='bads',
```
following the API of https://mne.tools/stable/generated/mne.pick_types.html for example.
Also do not forget to update the docstring. If should be a reflex. When you touch the signature of a public function you must update the docstring accordingly.

Reply:
In this case, I assumed `('bads')` would be equivalent to `'bads'` since a set of one string is still passed as just the string.

Reply:
`'bads'` and `('bads')` are indeed treated the same (but the latter is not a "set", that would be `{'bads'}`).  Removing the parentheses when they're not needed is done for readability (nobody will accidentally think it was a tuple or wonder if it was meant to be a tuple) so it makes the code easier to maintain.

Reply:
Ahh, Fair enough.